app/component/ItineraryTab.js

```diff
@@ -157,13 +157,23 @@ export default Relay.createContainer(ItineraryTab, {
           route {
             shortName
             gtfsId
+            longName
+            agency {
+              phone
+            }
           }
           trip {
             gtfsId
             tripHeadsign
             pattern {
               code
             }
+            stoptimes {
+              pickupType
```

Comment:
Should we add this directly to the leg information for the departure and destination stops?